### ISSUE TITLE
No docker images for recent versions

### ISSUE THREAD

**Issue by mjpieters** [pact-foundation/pact-reference#446]
The last version of the [pact-ref-verifier image on the docker hub](https://hub.docker.com/r/pactfoundation/pact-ref-verifier/tags) is 1.1.1. Can versions 1.1.2 and 1.1.3 be pushed there please?

Are there plans to push to the Github registry too, like is done for the stub server?

**Comment by YOU54F** [comment]
yes i’d like to add auto publishing to both dockerhub and ghcr, will look to add in next week

**Comment by rholshausen** [comment]
in the mean time I have pushed 1.1.2 and 1.1.3 up to docker hub.

**Comment by YOU54F** [comment]
thanks Ron!
